test-cases/manifest.ttl

```diff
@@ -1,3338 +1,4844 @@
-@base <http://w3id.org/rml/core/test-cases/id/> .
 @prefix dcterms: <http://purl.org/dc/terms/> .
+@prefix rml: <http://w3id.org/rml/> .
 @prefix rmltest: <http://w3id.org/rml/test/> .
 @prefix test: <http://www.w3.org/2006/03/test-description#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
-<RMLTC0000-CSV> a test:TestCase;
+<http://w3id.org/rml/core/test/RMLTC0000-CSV> a test:TestCase;
   dcterms:identifier "RMLTC0000-CSV";
-  rmltest:hasExpectedOutput true;
-  rmltest:input [
-      rmltest:inputFile "student.csv";
-      rmltest:inputType "CSV"
-    ];
+  rmltest:hasError false;
+  rmltest:input <http://w3id.org/rml/core/test/RMLTC0000-CSV/input/http%3A%2F%2Fw3id.org%2Frml%2Fresources%2Frml-io%2FRMLTC0000-CSV%2FFriends.json/>;
```

Comment:
something wrong with the link here.

Reply:
Oh right, the test data is modeled differently compared to rml-io. I fixed some of these already, but missed some I think.